PyZAPI/pyzapi/views/notfound.py

```diff
@@ -0,0 +1,7 @@
+from pyramid.view import notfound_view_config
+
+
+@notfound_view_config(renderer='../templates/404.jinja2')
```

Comment:
are we returning an html 404 here or a proper 404 resource not found rest response?